go.mod

```diff
@@ -73,7 +73,7 @@ require (
 	github.com/jmespath/go-jmespath v0.4.0
 	github.com/json-iterator/go v1.1.12
 	github.com/jung-kurt/gofpdf v1.16.2
-	github.com/lib/pq v1.10.7
+	github.com/lib/pq v1.10.6
```

Comment:
I would suggest adding a comment to prevent future attempts to upgrade until this is fixed

Reply:
Thanks @papagian added a comment at the bottom. Is that okay?